fix: bottom sheet height logic (#2859)
razorpay/blade: .changeset/orange-chefs-kick.md

---
'@razorpay/blade': minor
---

fix(blade): bottom sheet height logic

> > **Note:** This release introduces a change in height of  bottom sheet, earlier we were doing it wrong. we were adding header + grabhandle height, even if header don't have title or subtitle .

Reviewer: ```suggestion
> **Note:** This release introduces a change in height of  bottom sheet, earlier we were adding header + grabhandle height, even if header don't have title or subtitle. This release fixes that. 
```
